opensearch/src/main/java/org/opensearch/sql/opensearch/client/OpenSearchClient.java

```diff
@@ -22,6 +22,20 @@ public interface OpenSearchClient {
 
   String META_CLUSTER_NAME = "CLUSTER_NAME";
 
+  /**
+   * Check if the given index exists.
+   * @param indexName index name
+   * @return true if exists, otherwise false
+   */
+  boolean isExist(String indexName);
```

Comment:
```suggestion
  boolean exists(String indexName);
```

would be more conventional in Java, I think.

Reply:
yeah, `if table.exists` looks more natural. fixed. thanks!